topics/proteomics/tutorials/metaquantome-function/workflows/main_workflow.ga

```diff
@@ -0,0 +1,623 @@
+            "tool_version": null,
+            "type": "tool",
+            "uuid": "bd34e228-ef6a-4283-870b-2a14e1a2e06c",
+            "workflow_outputs": [
+                {
+                    "label": null,
+                    "output_name": "plotdata",
+                    "uuid": "0f5024a8-9d9f-4f71-b1d6-06a9da6468fa"
+                },
+                {
+                    "label": null,
+                    "output_name": "wrapped_outfile",
+                    "uuid": "44ecd108-ecf6-43b1-b669-ce30d6c4fc94"
+                }
+            ]
+        },
+        "11": {
+            "annotation": "",
+            "content_id": "toolshed.g2.bx.psu.edu/repos/galaxyp/metaquantome_viz/metaquantome_viz/2.0.0-0",
+            "errors": null,
+            "id": 11,
+            "input_connections": {
+                "infile": {
+                    "id": 6,
+                    "output_name": "outfile"
+                },
+                "samps": {
+                    "id": 1,
+                    "output_name": "samples_file"
+                }
+            },
+            "inputs": [
+                {
+                    "description": "runtime parameter for tool metaQuantome: visualize",
+                    "name": "infile"
+                },
+                {
+                    "description": "runtime parameter for tool metaQuantome: visualize",
+                    "name": "samps"
+                }
+            ],
+            "label": "T4-T2-Volcano-Plot",
+            "name": "metaQuantome: visualize",
+            "outputs": [
+                {
+                    "name": "wrapped_outfile",
+                    "type": "html"
+                },
+                {
+                    "name": "plotdata",
+                    "type": "tabular"
+                }
+            ],
+            "position": {
+                "left": 1504.4792175292969,
+                "top": 530.5382137298584
+            },
+            "post_job_actions": {},
+            "tool_id": "toolshed.g2.bx.psu.edu/repos/galaxyp/metaquantome_viz/metaquantome_viz/2.0.0-0",
+            "tool_shed_repository": {
+                "changeset_revision": "02da24eac445",
+                "name": "metaquantome_viz",
+                "owner": "galaxyp",
+                "tool_shed": "toolshed.g2.bx.psu.edu"
+            },
+            "tool_state": "{\"height\": \"\", \"infile\": {\"__class__\": \"RuntimeValue\"}, \"mode_args\": {\"mode\": \"f\", \"__current_case__\": 0, \"ontology\": \"go\", \"plot\": {\"plottype\": \"volcano\", \"__current_case__\": 1, \"fc_name\": \"log2fc_T4_over_T2\", \"fc_corr_p\": \"corrected_p_T4_over_T2\", \"textannot\": \"id\", \"flip_fc\": \"false\", \"gosplit\": \"false\"}}, \"samps\": {\"__class__\": \"RuntimeValue\"}, \"width\": \"\", \"__page__\": null, \"__rerun_remap_job_id__\": null}",
```

Comment:
```suggestion
            "tool_state": "{\"height\": \"\", \"infile\": {\"__class__\": \"RuntimeValue\"}, \"mode_args\": {\"mode\": \"f\", \"__current_case__\": 0, \"ontology\": \"go\", \"plot\": {\"plottype\": \"volcano\", \"__current_case__\": 1, \"fc_name\": \"log2fc_T4_over_T2\", \"fc_corr_p\": \"corrected_p_T4_over_T2\", \"textannot\": \"id\", \"flip_fc\": \"false\", \"gosplit\": \"true\"}}, \"samps\": {\"__class__\": \"RuntimeValue\"}, \"width\": \"\", \"__page__\": null, \"__rerun_remap_job_id__\": null}",
```

Reply:
"Make one plot for each of BiologicalProcess, CellularComponent, and MolecularFunction GO term categories." is set to "Yes" according to the tutorial, so please update the `gosplit` parameter in the workflow.